libs/template/cli/src/commands/create.spec.ts

```diff
@@ -0,0 +1,467 @@
+import * as prompts from '@clack/prompts';
+import fs, { PathLike } from 'fs';
+import path from 'path';
+import { expect, it, vi } from 'vitest';
```

Comment:
We use global in tscofig. So exports are not necessary. I think same for other files

Reply:
I need at least `vi` and `expect` imports for the typings to be correct, `it` can be removed.